app/gui/src/project-view/composables/animation.ts

```diff
@@ -158,24 +160,26 @@ function useApproachBase<T>(
 
 /** TODO: Add docs */
 export function useTransitioning(observedProperties?: Set<string>) {
-  const hasActiveAnimations = ref(false)
+  const hasActiveTransitions = ref(false)
+  const animCounter = provideAnimationCounter()
```

Comment:
We don't increase counter here. 

If we shouldn't, then I have no idea what is counted here - docs are strongly advised.

Reply:
This is the provide end. We are using the counter on this side, the counting is done in the size animation component.

Reply:
This is the provide end. We are using the counter on this side, the counting is done in the size animation component. I'll add docs to the store.